### PR TITLE
replace gasPrice and gasprice in $ calculation for avax

### PR DESCRIPTION
**Pull request by 0xKirkjufell**
image

Dapp change checklist:
  - [ ] Code reviewed by 1 reviewers.
  - [x] CI passes

### CODE REVIEW

File: src/hooks/useSwapEstimator.js [hunk 1]
```diff
@@ -495,33 +473,63 @@ const useSwapEstimator = ({
 
   // Fetches current gas price
   const fetchGasPrice = async () => {
+    // https://docs.avax.network/learn/platform-overview/transaction-fees/#dynamic-fee-transactions
+    let jsonCallId = 1
+    let gasPriorityFee = BigNumber.from(0)
+    const nAVAX = '000000000'
+    let gasPrice = BigNumber.from('25' + nAVAX)
+
+    const fetchMethodResult = async (method) => {
```
Comment (0xKirkjufell) [inline src/hooks/useSwapEstimator.js:482]: Structurally, similar method is also used in accountlistener.js to fetch balances. I am not sure if this should be moved to util. Thoughts?

File: src/hooks/useSwapEstimator.js [hunk 2]
```diff
@@ -495,33 +473,63 @@ const useSwapEstimator = ({
...
 
       if (!isGasPriceUserOverriden) {
         ContractStore.update((s) => {
           s.gasPrice = gasPrice
         })
       }
-      return gasPrice
     } catch (e) {
       console.error(
-        `Can not fetch gas prices, using chainlink as fallback method: ${e.message}`
```
Comment (0xKirkjufell) [inline src/hooks/useSwapEstimator.js:519]: To be clear about reasoning: there is not chainlink gas price. We could use a 3rd party API for the gas price to keep parity with 2-method attempt, but I think given lower volatility of avax gas (it's almost always ~25 nAVAX), that we are OK.

File: src/hooks/useSwapEstimator.js [hunk 3]
```diff
@@ -441,7 +441,7 @@ const useSwapEstimator = ({
   const _fetchEthPriceCryptoApi = async () => {
```
Comment (orph) [inline src/hooks/useSwapEstimator.js:441]: Rename Eth -> Avax while here? Or remove if not needed

File: src/hooks/useSwapEstimator.js (hunk 1 above)
Comment (orph) [reply src/hooks/useSwapEstimator.js:482]: Moving makes sense as this is standalone - up to you

File: src/hooks/useSwapEstimator.js [hunk 4]
```diff
@@ -457,28 +457,6 @@ const useSwapEstimator = ({
...
-      }
-      return priceFeed.answer
-    } catch (e) {
-      console.error('Error happened fetching fast gas chainlink data:', e)
-    }
-
-    return BigNumber.from(0)
-  }
-
   const _fetchEthPriceChainlink = async () => {
```
Comment (orph) [inline src/hooks/useSwapEstimator.js:482]: Remove?

**Review by orph — APPROVED** [review]
Minor requests while here

**Pull request closed without merging**